modules/InfiniteStories/ui/CategoriesFilters/CategoriesFilters.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Category } from '@prezly/sdk/dist/types';
+import type { Locale } from '@prezly/theme-kit-nextjs';
+import classNames from 'classnames';
+import Link from 'next/link';
+
+import { PageTitle } from '@/components/PageTitle';
+
+import styles from './CategoriesFilters.module.scss';
+
+interface Props {
+    activeCategory: Pick<Category, 'id'> | undefined;
+    categories: Category[];
+    className?: string;
+    locale: Locale.Code;
+}
+
+export function CategoriesFilters({ activeCategory, categories, className, locale }: Props) {
+    return (
+        <div className={className}>
+            <PageTitle className={styles.title} title="Latest stories" />
```

Comment:
I'll have to request translations for these two strings, but for now we can deploy this as-is since users do not have access to featured categories in the app yet.